tests/async/test_async_auth_decorators.py

```diff
@@ -0,0 +1,145 @@
+from functools import partial

```

Comment:
Please move these tests to the `tests/decorators` as we did for other decorators.

Reply:
Please also add two "standard" tests that we've added for all decorators with async views support: `test_wrapped_sync_function_is_not_coroutine_function`, `test_wrapped_async_function_is_coroutine_function` (e.g. 3152f9de47a317d658ebee6d5b517b7e1258aa1f).